 - Fix up default paths
 - Support env CC
 - Don't build/install libtz.a
 - Don't install man-pages provided by sys-apps/man-pages
 - Move tzselect, zic, and zdump to bin
 - Do not run network based unittests

--- a/Makefile	2016-03-27 13:21:07.021182166 +0700
+++ b/Makefile	2016-03-27 13:21:12.499246226 +0700
@@ -44,21 +44,21 @@
 
 # Everything gets put in subdirectories of. . .
 
-TOPDIR=		/usr/local
+TOPDIR=		/usr
 
 # "Compiled" time zone information is placed in the "TZDIR" directory
 # (and subdirectories).
 # Use an absolute path name for TZDIR unless you're just testing the software.
 
 TZDIR_BASENAME=	zoneinfo
-TZDIR=		$(TOPDIR)/etc/$(TZDIR_BASENAME)
+TZDIR=		$(TOPDIR)/share/$(TZDIR_BASENAME)
 
 # Types to try, as an alternative to time_t.  int64_t should be first.
 TIME_T_ALTERNATIVES= int64_t int32_t uint32_t uint64_t
 
 # The "tzselect", "zic", and "zdump" commands get installed in. . .
 
-ETCDIR=		$(TOPDIR)/etc
+SBINDIR=		$(TOPDIR)/sbin
 
 # If you "make INSTALL", the "date" command gets installed in. . .
 
@@ -66,7 +66,7 @@
 
 # Manual pages go in subdirectories of. . .
 
-MANDIR=		$(TOPDIR)/man
+MANDIR=		$(TOPDIR)/share/man
 
 # Library functions are put in an archive in LIBDIR.
 
@@ -339,8 +339,10 @@
 
 #MAKE=		make
 
-cc=		cc
-CC=		$(cc) -DTZDIR=\"$(TZDIR)\"
+CPPFLAGS+=		-DTZDIR=\"$(TZDIR)\"
+ifeq ($(NLS),1)
+CPPFLAGS+= -DHAVE_GETTEXT=1 -DTZ_DOMAIN=\"libc\"
+endif
 
 AR=		ar
 
@@ -391,17 +393,13 @@
 
 install:	all $(DATA) $(REDO) $(MANS)
 		mkdir -p $(DESTDIR)$(ETCDIR) $(DESTDIR)$(TZDIR) \
-			$(DESTDIR)$(LIBDIR) \
+			$(DESTDIR)$(BINDIR) \
 			$(DESTDIR)$(MANDIR)/man3 $(DESTDIR)$(MANDIR)/man5 \
 			$(DESTDIR)$(MANDIR)/man8
 		$(ZIC_INSTALL) -l $(LOCALTIME) -p $(POSIXRULES)
 		cp -f iso3166.tab $(ZONETABLES) $(DESTDIR)$(TZDIR)/.
-		cp tzselect zic zdump $(DESTDIR)$(ETCDIR)/.
-		cp libtz.a $(DESTDIR)$(LIBDIR)/.
-		$(RANLIB) $(DESTDIR)$(LIBDIR)/libtz.a
+		cp tzselect zic zdump $(DESTDIR)$(BINDIR)/.
 		cp -f newctime.3 newtzset.3 $(DESTDIR)$(MANDIR)/man3/.
-		cp -f tzfile.5 $(DESTDIR)$(MANDIR)/man5/.
-		cp -f tzselect.8 zdump.8 zic.8 $(DESTDIR)$(MANDIR)/man8/.
 
 INSTALL:	ALL install date.1
 		mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
@@ -496,7 +494,7 @@
 		chmod +x $@
 
 check:		check_character_set check_white_space check_links check_sorted \
-		  check_tables check_web
+		  check_tables
 
 check_character_set: $(ENCHILADA)
 		LC_ALL=en_US.utf8 && export LC_ALL && \
